### DISCUSSION TITLE
工作流Failure

### DISCUSSION THREAD

**Discussion by Kzused** [lxysl/Auto_Attendance#25] [Q&A]
作者你好，我想问问工作流几天之前可以正常打卡，但是现在突然就无法打卡了，为什么会这样？
![image](https://user-images.githubusercontent.com/52688044/117751049-07e80380-b247-11eb-8ab4-777d4f3003d2.png)

**Comment by Mincana-Huang** [comment]
同样的问题，查看工作流提示无法登录

**Comment by lxysl** [comment]
你好，请看最新的issue
